components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,244 @@
+      optional: true,
+    },
+    supportsAllDrives: {
+      type: "boolean",
+      label: "Supports All Drives",
+      description:
+        "Whether to include shared drives. Set to 'true' if saving to a shared drive. Defaults to 'false' if left blank.",
+      optional: true,
+    },
+    contentHintsIndexableText: {
+      type: "string",
+      label: "Content Hints Indexable Text",
+      description:
+        "Text to be indexed for the file to improve fullText queries. This is limited to 128KB in length and may contain HTML elements.",
+      optional: true,
+    },
+    contentRestrictionsReadOnly: {
+      type: "boolean",
+      label: "Content Restrictions Read Only",
+      description:
+        "Whether the content of the file is read-only. If a file is read-only, a new revision of the file may not be added, comments may not be added or modified, and the title of the file may not be modified.",
+      optional: true,
+    },
+    contentRestrictionsReason: {
+      type: "string",
+      label: "Content Restrictions Reason",
+      description:
+        "Reason for why the content of the file is restricted. This is only mutable on requests that also set readOnly=true.",
+      optional: true,
+    },
+    copyRequiresWriterPermission: {
+      type: "boolean",
+      label: "Copy Requires Writer Permission",
+      description:
+        "Whether the options to copy, print, or download this file, should be disabled for readers and commenters.",
+      optional: true,
+    },
+    description: {
+      type: "string",
+      label: "Description",
+      description: "A short description of the file.",
+      optional: true,
+    },
+    folderColorRgb: {
+      type: "string",
+      label: "Folder Color RGB",
+      description:
+        "The color for a folder as an RGB hex string. If an unsupported color is specified, the closest color in the palette will be used instead.",
+      optional: true,
+    },
+    mimeType: {
+      type: "string",
+      label: "Mime Type",
+      description: `The MIME type of the file. Google Drive will attempt to automatically detect an appropriate value from uploaded content if no value is provided. The value cannot be changed unless a new revision is uploaded. If a file is created with a Google Doc MIME type, the uploaded content will be imported if possible. 
+        Google Workspace and Drive MIME Types: https://developers.google.com/drive/api/v3/mime-types
+      `,
+      optional: true,
+    },
+    name: {
+      type: "string",
+      label: "Name",
+      description: "Name of the file",
+      optional: true,
+    },
+    originalFilename: {
+      type: "string",
+      label: "Original Filename",
+      description:
+        "The original filename of the uploaded content if available, or else the original value of the name field. This is only available for files with binary content in Google Drive.",
+      optional: true,
+    },
+    parents: {
```

Comment:
Just for the sake of UX, I'd move this options to the top since they are more relevant to the user setting up the action: we want users to get going as fast as possible, and each additional prop that they have to configure gets them one step closer to quitting the whole thing in anger 😂 

As a reference, Zapier only provides a handful of very important props for users to configure and actually hides the rest of the options from users:
![image](https://user-images.githubusercontent.com/5479513/128447647-9c4a8a41-07d3-48c3-bdf5-ca28a1ee8b49.png)

Reply:
Moved parents prop up